nixos/modules/services/web-servers/apache-httpd/mediawiki.nix

```diff
@@ -305,10 +305,10 @@ in
 
   # !!! Need to specify that Apache has a dependency on PostgreSQL!
 
-  startupScript = pkgs.writeScript "mediawiki_startup.sh"
+  startupScript = pkgs.writeScript "mediawiki_startup.sh" ''
     # Initialise the database automagically if we're using a Postgres
     # server on localhost.
-    (optionalString (config.dbType == "postgres" && config.dbServer == "") ''
+    ${optionalString (config.dbType == "postgres" && config.dbServer == "") ''
```

Comment:
Now you're embedding a comment in the script that might have no code following it? Maybe move line 311 to line 309?